[release] v7.6.2 (#13395)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 7.7.0

_Jun 6, 2024_

We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:

> - ✨ Introduced plugins system in the Charts @alexfauquette

Reviewer: @alexfauquette we probably want to leave this out for now? 😅

Author: out of the highlights? or generally out of the CHANGELOG?

Reviewer 2: AFAIK it's purely an internal thing so our of the highlights

Reviewer 2: And the release can probably be a patch, I guess this is the PR that justified the minor for you :+1:

- 🐞 Bugfixes
- 📚 Documentation improvements

<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->

### Date and Time Pickers

#### `@mui/x-date-pickers@7.7.0`

> - [fields] Fix new index calculation on letter section (#13310) @arthurbalduini

Reviewer 2: ```suggestion
- [fields] Fix `PageUp` and `PageDown` editing on letter sections (#13310) @arthurbalduini
```

> - [pickers] Fix `AdapterDayjs` timezone behavior (#13362) @LukasTy

Reviewer 2: ```suggestion
- [pickers] Fix `AdapterDayjs` timezone behavior (#13362) @LukasTy
- [pickers] Use `useRtl` instead of `useTheme` to access direction (#13363) @flaviendelangle 
```

> - 

Reviewer 3: ```suggestion
```

#### `@mui/x-date-pickers-pro@7.7.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@7.7.0`.

### Charts

#### `@mui/x-charts@7.7.0`

- [charts] Add `Initializable` type and behaviour to allow checking if a complex context has been initialized. (#13365) @JCQuintas

> - [charts] Fix some props not working in xAxis/yAxis (#13372) @Valyok26

Reviewer 2: ```suggestion
- [charts] Fix some props not working in `xAxis` and `yAxis` (#13372) @Valyok26
```

> - [charts] Harmonize charts Types (#13366) @alexfauquette

Reviewer 2: ```suggestion
- [charts] Harmonize charts types (#13366) @alexfauquette
```

- [charts] Introduce plugins system (#13367) @alexfauquette

### Docs

- [docs] Add badges like in Material UI @oliviertassinari
- [docs] Update twitter.com to x.com @oliviertassinari
- [docs] Fix the description of `tickInterval` (#13355) @alexfauquette

> - [docs] adjusted the code example for `quickFilterValues` (#12919) @michelengelen

Reviewer 3: ```suggestion
- [docs] Adjust the code example for `quickFilterValues` (#12919) @michelengelen
```

### Core

- [core] Comment on `CSS.escape` for the future @oliviertassinari
- [core] Fix `l10n` action setup (#13382) @LukasTy
- [core] Fixes in preparation for React 18.3 (#13378) @LukasTy
- [core] Remove explicit `marked` dependency (#13383) @LukasTy
- [core] Remove unused `@types/prettier` dependency (#13389) @LukasTy
- [core] add docs .env.local to .gitignore (#13377) @KenanYusuf

> - [data grid] Add the `areElementSizesEqual` utility to improve code readability (#13254) @layerok

Reviewer 2: Should be in the grid and with the correct prefix :+1:

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 7.7.0

_Jun 6, 2024_

We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:

- 🐞 Bugfixes
- 📚 Documentation improvements

<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->

### Data Grid

#### `@mui/x-data-grid@7.70`

> - [data grid] Add the `areElementSizesEqual` utility to improve code readability (#13254) @layerok

Reviewer 3: ```suggestion
- [DataGrid] Add the `areElementSizesEqual` utility to improve code readability (#13254) @layerok
```

- 
#### `@mui/x-date-pickers-pro@7.6.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@7.6.2`.

### Charts

#### `@mui/x-charts@7.6.2`

- [charts] Add `Initializable` type and behaviour to allow checking if a complex context has been initialized. (#13365) @JCQuintas
- [charts] Fix some props not working in `xAxis` and `yAxis` (#13372) @Valyok26
- [charts] Harmonize charts types (#13366) @alexfauquette
- [charts] Introduce plugins system (#13367) @alexfauquette

### Docs

- [docs] Add badges like in Material UI @oliviertassinari
- [docs] Update twitter.com to x.com @oliviertassinari
- [docs] Fix the description of `tickInterval` (#13355) @alexfauquette
- [docs] adjusted the code example for `quickFilterValues` (#12919) @michelengelen

### Core

- [core] Comment on `CSS.escape` for the future @oliviertassinari
- [core] Fix `l10n` action setup (#13382) @LukasTy
- [core] Fixes in preparation for React 18.3 (#13378) @LukasTy
- [core] Remove explicit `marked` dependency (#13383) @LukasTy
- [core] Remove unused `@types/prettier` dependency (#13389) @LukasTy

> - [core] add docs .env.local to .gitignore (#13377) @KenanYusuf

Reviewer 3: ```suggestion
- [core] Add `docs/.env.local` to `.gitignore` (#13377) @KenanYusuf
```

Same changes as in `@mui/x-data-grid-pro@7.6.2`.

### Date and Time Pickers

#### `@mui/x-date-pickers@7.6.2`

- [fields] Fix `PageUp` and `PageDown` editing on letter sections (#13310) @arthurbalduini
- [pickers] Fix `AdapterDayjs` timezone behavior (#13362) @LukasTy
- [pickers] Use `useRtl` instead of `useTheme` to access direction (#13363) @flaviendelangle

#### `@mui/x-date-pickers-pro@7.6.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@7.6.2`.

### Charts

#### `@mui/x-charts@7.6.2`

- [charts] Add `Initializable` type and behaviour to allow checking if a complex context has been initialized. (#13365) @JCQuintas
- [charts] Fix some props not working in `xAxis` and `yAxis` (#13372) @Valyok26
- [charts] Harmonize charts types (#13366) @alexfauquette
- [charts] Introduce plugins system (#13367) @alexfauquette

### Docs

- [docs] Add badges like in Material UI @oliviertassinari
- [docs] Update twitter.com to x.com @oliviertassinari
- [docs] Fix the description of `tickInterval` (#13355) @alexfauquette
- [docs] Adjust the code example for `quickFilterValues` (#12919) @michelengelen

> - [docs] Create pickers accessibility page (#13274) @arthurbalduini

Reviewer 4: This could probably be a highlight?

Reviewer 5: This is debatable, but we tend to use `Pickers` with uppercase `P` when referring to the product. 🤔 
```suggestion
- [docs] Create Pickers accessibility page (#13274) @arthurbalduini
```

Reviewer 2: I'd even argue for "Date and Time Pickers" :laughing:
